variables.tf

```
@@ -11,9 +11,9 @@ variable "name" {
 }
 
 variable "cidr" {
-  description = "The CIDR block for the VPC. Default value is a valid CIDR, but not acceptable by AWS and should be overridden"
+  description = "(Optional) The IPv4 CIDR block for the VPC. CIDR can be explicitly set or it can be derived from IPAM using `ipv4_netmask_length` & `ipv4_ipam_pool_id`"
   type        = string
-  default     = "0.0.0.0/0"
+  default     = null
```

we can keep this default

```suggestion
  default     = "0.0.0.0/0"
```

---

can i ask why? with the changes i mentioned above `null` seems to be a better fit